spec/requests/api/v1/authentication_controller/forgot_password_spec.rb

```diff
@@ -20,11 +20,12 @@
         # creates a test user
         user = FactoryBot.create(:user)
         post "/api/v1/password/forgot", params: { email: user.email }, as: :json
+        @user = user
       end
 
       it "returns status 200 and should send reset password instructions" do
         expect(response).to have_http_status(:ok)
-        expect(Devise::Mailer).to send_email(:reset_password_instructions)
+        expect(response.parsed_body["message"]).to eq("password reset instructions sent to #{@user.email}")
```

Comment:
Remove this change as it is not related and false positive

Reply:
This will have to rebased after service spec PR is merged as I have changed the logic of service... then will refactor the service component(method). Also would have to add one test case.